body.tex

```diff
@@ -692,7 +699,95 @@ \subsection{Reporting Cycle}
+
+Standard terminology for financial reserves is that ``management reserve'' describes funds held back to pay for the unknown unknowns, while ``contingency reserve'' is used for known unknowns (i.e.\ the risks in the risk register).
+Rubin Operations' two funding agencies offer differing guidelines on how reserve should be managed.
+DOE allows reserves to be built up and carried forward at SLAC, and used as both contingency and management reserve.
+DOE HEP holds additional reserves centrally, which can also be used as both contingency and management reserve.
+NSF AST allows line item budget over-estimation by up to 10%, to allow for increases in cost.
+These generate carry-forward (along with any other under-spending that may occur), which we use as contingency reserve.
+NSF also holds reserves of both kinds centrally, to cover high cost impact realized risks as well as unexpected events.
+
+The cost impact analysis of the Rubin Operations risk register informs the level of contingency reserve needed.
+Residual (post-mitigation) cost exposure for each risk is computed by taking the  product of the residual probability of the risk being realized (in any given year) and the likely annual cost of addressing that risk.
+The total cost exposure is then the sum, over all risks in the register, of these products.
+
+Currently, the total cost exposure, and hence the needed level of contingency reserve, is approximately 15\% of the total annual operating budget at each operations partner.
+Included in the risk register is the risk of federal funding allocations being reduced, with a likely cost (if realized in any given year) of some \$1.5M per operations partner.
+(At SLAC, for example, this would cover about 7\% of the operating expense -- enough to pay the labor bills during a 1 month funding allocation delay.)
+
+Reserve maintenance is accounted for in our federal budget requests, either as an explicit difference between cost and budget on the DOE side, or as line by line allowances for increases in cost on the NSF side.
+At both operations partners the reserve is maintained as carry-forward from one year's spending to the next, with it being spent down to zero either during the post-operations period (at SLAC), or at the end of each 5-year renewal period at NOIRLab.
+(The restriction on carrying forward funds between renewal periods means that the level of reserve held at NOIRLab varies significantly over the course of the 5 years.
+Risks that are realized early in the 5-year period are more likely to need supplemental funding from MSF's centrally-held reserves than those that are realized later on, when the carry-forward has been built up.
+It may be possible to purposefully hold back scope so as to build up the carry-forward quickly at the start of a renewal period.)
+
+Requests beyond target (RBTs) are typically made by the departments in order to address risks as they are realized.
```

Comment:
"...address risks and/or opportunities as ..."

We probably need more on opportunities generally. But can wait for later.

Reply:
Modification made. Need to think about opportunity registration, agreed.